tests/dummy/app/templates/bind-test.hbs

```diff
@@ -9,3 +9,5 @@
 <div class="test5">{{data-test-component data-test="foo"}}</div>
 
 <div class="test6">{{data-test-component data-non-test="foo"}}</div>
+
+<div class="test7">{{data-test-component data-test-without-value}}</div>
```

Comment:
Would a test case for `{{component data-test}}` similar to `.test5` we have now be helpful?